docs/troubleshooting-guide.md

```diff
@@ -0,0 +1,146 @@
+# Front-End Monorepo - Troubleshooting Guide
+
+At this moment in time - the distant past of October 2018 - the Zooniverse
+Front-End Monorepo is a rapidly fluctuating work in progress. New devs jumping
+in may have trouble figuring out how to even get started, so this evolving
+document was added to help with the weird errors and issues that may not yet be
+covered in the "proper" documentation.
+
+(@shaun.a.noordin 2018.10.12)
+
+NOTE: this evolving doc should be reviewed regularly and should be retired
+eventually. Any important points should be folded into the Monorepo's proper
+documentation.
+
+## Installing & Running Packages
+
+Do what the README.MD tells you:
+
+1. Clone the repo: `git clone git@github.com:zooniverse/front-end-monorepo.git`
+2. Run Lerna's bootstrap: `lerna bootstrap`
+  1. NOTE: this installs dependencies for every package, but be prepared for
+     jankiness as memory leaks might occur, files are opened for writing but
+     never closed, the OS lagging and etc. Doesn't always happen, but sometimes
+     it does.
+  2. NOTE: Despite the README saying that you should first enter into the
```

Comment:
You can add a `--scope` parameter to the call so that it only runs for the package that you're working in, thus:
`cd package/lib-classifier`
`lerna bootstrap --scope=@zooniverse/classifier`

Note the scope should match what the name is in package.json. The scope param also works for the `lerna add` command, so if you want to say add the react-component package, but only to the current package you're working on, you use the scope param:

`lerna add @zooniverse/react-components --scope=@zooniverse/lib-classifier`

Reply:
Alright, will try `--scope` in my next iteration 👍 If that works as a consistent solution I'll update the main README with the info.